tools/check-enforcer/Azure.Sdk.Tools.CheckEnforcer/GitHubWebhookProcessor.cs

```diff
@@ -49,13 +49,26 @@ public async Task ProcessWebhookAsync(string eventName, string json, ILogger log
         {
             await Policy
                 .Handle<AbuseException>()
+                .Or<RateLimitExceededException>()
```

Comment:
Do all other exceptions end up with the item back on the queue and retried immediately? If so, seems ideal we have a back off for those cases as well?

Reply:
No, exceptions that do not appear in that list of Handle<type>().Or<type>() don't get retried. They'll fail immediately (based on my understanding of the way that the Polly library works.